src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +794,76 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        deployment_configs = self.list_deployment_configs()
+
+        data = {"Config Name": [], "Instance Type": []}
+
+        for deployment_config in deployment_configs:
+            if deployment_config.get("DeploymentConfig") is None:
+                continue
+
+            benchmark_metrics = deployment_config.get("BenchmarkMetrics")
+
+            data["Config Name"].append(deployment_config.get("ConfigName"))
+            data["Instance Type"].append(
+                deployment_config.get("DeploymentConfig").get("InstanceType")
+            )
+
+            for benchmark_metric in benchmark_metrics:
+                column_name = f"{benchmark_metric.get('name')} ({benchmark_metric.get('unit')})"
+                if column_name in data:
+                    data[column_name].append(benchmark_metric.get("value"))
+                else:
+                    data[column_name] = [benchmark_metric.get("value")]
+
+        df = pd.DataFrame(data)
+        # Temporarily print markdown
+        print(df.to_markdown())
+
+    def list_deployment_configs(self) -> list[dict[str, Any]]:
+        """List deployment configs for ``This`` model in the current region."""
```

Comment:
Let's update comment here to indicate what the output of function.

https://peps.python.org/pep-0257/